sql/api/src/main/scala/org/apache/spark/sql/streaming/StatefulProcessorHandle.scala

```diff
@@ -38,6 +38,18 @@ private[sql] trait StatefulProcessorHandle extends Serializable {
    */
   def getValueState[T](stateName: String): ValueState[T]
 
+  /**
+   * Creates new or returns existing list state associated with stateName.
+   * The ListState persists values of type T.
+   *
+   * @param stateName  - name of the state variable
+   * @param keyEncoder - Spark SQL Encoder for key
```

Comment:
Should we update this ?

Reply:
yeah, removed the docs for removed argument.